src/main/java/dominio/Review.java

```diff
@@ -0,0 +1,83 @@
+package dominio;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Random;
+import java.util.UUID;
+
+/*
+ * Classe que possui a avaliação de um item de pedido de algum livro adquirido por um cliente
+ */
+public class Review implements Serializable {
+	private static final long serialVersionUID = -4063511252485472431L;
+	
+	private final String id;
+	
+	private final Customer customer;
+	
+	private final Book book;
+	
+	private final int bookstoreId;
+	
+	private double value;
+	
+	public Review(String id, Customer customer, Book book, double value, int bookstoreId) throws IOException
+	{
+		this.id = id;
+		
+		this.customer = customer;
+		
+		this.book = book;
+		
+		this.bookstoreId = bookstoreId;
+		
+		this.setValue(value);
+	}
+	
+	public Review(Customer customer, Book book, double value, int bookstoreId) throws IOException
+	{
+		this.id = UUID.randomUUID().toString();
+		
+		this.customer = customer;
+		
+		this.book = book;
+		
+		this.value = value;
+		
+		this.bookstoreId = bookstoreId;
+		
+		this.validateEvaluateValue();
```

Comment:
```suggestion

		this.bookstoreId = bookstoreId;
		this.rating=setRating(rating)
```

ou no caso value se não quiser mudar